src/components/Annotations/Rectangle.tsx

```diff
@@ -1,33 +1,43 @@
 import { SVGProps } from 'react';
 
-import { usePositionAndSize } from '../../hooks';
+import { useRectanglePosition } from '../../hooks';
 
 export interface AnnotationRectangleProps
   extends Omit<
     SVGProps<SVGRectElement>,
     'x1' | 'x2' | 'y1' | 'y2' | 'x' | 'y' | 'width' | 'height'
```

Comment:
It seems to me that we should also omit 'stroke' because the use of `color` will be mandatory. Same is true for all the other elements in which we are using `color` instead of `fill` or `stroke`.  @targos WDYT ?

Reply:
@lpatiny Do you want me to remove them?